tests/_support/Step/Acceptance/Admin/CampaignGeneratorSteps.php

```diff
@@ -0,0 +1,51 @@
+<?php
+namespace Step\Acceptance\Admin;
+
+/**
+ * Gutenberg: steps to use Gutenberg editor
+ */
+class CampaignGeneratorSteps
+{
+	/**
+	 * @var \AcceptanceTester;
+	 */
+	protected $tester;
+
+	public function __construct(
+		\AcceptanceTester $I
+	) {
+		$this->tester = $I;
+	}
+
+	/**
+	 * @Then I see Analytics & Tracking section validation error message
+	 */
+	public function iSeeAnalyticsTrackingSectionValidationErrorMessage()
+	{
+		$I = $this->tester;
+		$I->waitForText( 'Please check "Analytics & Tracking" section for required fields.', 5 );
+		$I->see( 'Global Project is a required field' );
+	}
+
+	/**
+	 * @Then I select global project :arg1
+	 */
+	public function iSelectGlobalProject( $arg1 )
+	{
+		$I = $this->tester;
+		$I->click( '//button[@aria-label="Close panel"]' );
+		$I->click( '//button[@data-label="Document"]' );
+		$I->appendField( '#p4_campaign_name', $arg1 );
+	}
+
+	/**
+	 * @Then the campaign is visible on the website
+	 */
+	public function theCampaignIsVisibleOnTheWebsite()
+	{
+		$I = $this->tester;
+		$I->click('View Campaign');
+		$I->see('Test campaign title');
```

Comment:
This is a bit too specific for the sentence `the campaign is visible on the website`, especially if I want to test another campaign. Maybe `the campaign is visible on the website with title :arg1 and paragraph :arg2` so that it is reusable ? 
The real solution would be to handle context, but it's probably something to add later (I'm sure there's a todo mention in there :p)